brave/browser-laptop
# Install / update extensions w/ component updater

Requires: 
https://github.com/brave/electron/pull/59
https://github.com/brave/libchromiumcontent/pull/4
https://github.com/brave/brightray/pull/1
https://github.com/brave/vault-updater/pull/33

Fix #4080
Fix #4081

Test Plan: 
- Completely retest each extension from scratch.
- Try cleaning your userProfileDir/Extensions but not the session-store-1 file, things should be recoverable / normal
- Try cleaning your session-store-1, things should be recoverable. 
- Try manually setting your session-store-1 for 1passwrod from 4.5.9.90 to 4.4.9.90 (replace all), edit `~/Library/Application\ Support/brave-development/Extensions/aomjjhallfgjeglblehebfpbcfeobpgk/4.5.9.90/manifest.json` and set the version inside the manifest to 4.4.9.90.  Move the folder to a subfolder called 4.4.9.90 as well.
- Start up and wait 10 minutes, it should update. Or you can run `require('electron').remote.session.defaultSession.updateClient.checkNow('aomjjhallfgjeglblehebfpbcfeobpgk')` to force the update.  about:extensions should show the new version.

Auditors: @bridiver, @darkdh
